### ISSUE TITLE
User research - final day

### ISSUE THREAD

**Issue by jacattell** [claussian/hh-typology#20]
# Suggestions from final user research

I've removed all the positive comments (of which there were many).
## Financial position and attitudes:
- Make all the text bigger, with no italics and no bold
- Make the [radio] buttons bigger
- Get rid of italics
- Change the title to, "Bank of England household survey - UK financial attitudes". Then the BoE name comes right next to the logo
- Make "Multidimensional Scaling" a hyperlink which opens a new window so I can read about it, maybe on wikipedia
- Change, "On this plot, each circle represents a household and households placed closer together have more similar answers to questions related to financial attitude and outlook" to "Each circle below represents a household. Circles placed closer together have similar answers to financial attitude and outlook survey questions"
- Explain the "index score" in the bottom left hand corner
- In "Certainty in house income a year from now" move "good idea" to the far right. Otherwise it reads like "its a good idea to have certainty a year from now"
- Make the key be bigger, ie Secure/Struggling
- Make "Perceived higher taxes due to govt budget measures" be on 1 line, instead of split over 2. Same for the chart title below it.
- Change "govt" to "government"
## Demographics:
- Use the same font everywhere, ie the map text to sans serif
- Get rid of italics
- In qualifications, explain the difference between "Other" and "School"
- In job status, make the order chronological, ie student, employed, unemployed, retired
- Make the gender key bigger, ie Struggling/Secure
- Put female before male

**Comment by claussian** [comment]
Changed title to Bank of England household survey - UK financial attitudes

**Comment by claussian** [comment]
Included hyperlink for Multidimensional Scaling to wikipedia

**Comment by claussian** [comment]
Changed main text above MDS plot to to "Each circle below represents a household. Circles placed closer together have similar answers to financial attitude and outlook survey questions"

**Comment by claussian** [comment]
Included text on income/debt score but pending review from Chris/James/James' contacts

**Comment by jacattell** [comment]
Yes, it reads better. Can you simplify it further? Entries will be displayed on the Bank of England intranet. Anyway to connect this to the crisis in the UK housing market will help. There is a shortage of affordable housing, especially in London. Investors have artificially inflated house prices, putting home ownership beyond the reach of many. Rental prices are also cripplingly high for many, with talk for rent control being introduced. As one London School of Economics Professor told me recently, "The number 1 thing that the next British Parliament must sort out is the UK housing market". Hope that helps

**Comment by jacattell** [comment]
I'll reach out to peeps I know for advice

**Comment by claussian** [comment]
Removed not applicable in nkids

**Comment by claussian** [comment]
Made colour scale for chloropleth map consistent across household types (always compute between 0-6%)

**Comment by claussian** [comment]
Changed labels for qualification
